lib/DataHandler.ts

```diff
@@ -74,6 +74,11 @@ export default class DataHandler {
       args: item.command.args,
     };
 
+    if (item.command.name == "ssubscribe" && err.message.includes("MOVED")) {
+      this.redis.emit("moved");
+      return;
+    }
+
```

Comment:
Can you please add a comment here explaining how we use the standalone client for `ssubscibe` connections

Reply:
When a hashslot is moved to another shard the message that is sent on the pubsub channel is `sunsubscribe`, not `moved`. I suspect this is working because it's caught when the connection is retried. On issuing the `ssubscribe` command, the response is `moved`.